llvm/utils/lit/tests/Inputs/shtest-env/env-args-none.txt

```diff
@@ -1 +1,30 @@
-# RUN: env
+# Check default environment.
+# RUN: env | FileCheck -check-prefix=NO-ARGS %s
+#
+# NO-ARGS: BAR = 2
```

Comment:
Did you actually run these tests, I can't see how this is passing?
It isn't using the `print_environment.py` script so there should be no spaces between the output. Additionally, I think we need to sort the variables as noted above to make this tests non-flaky.

Reply:
I will examine this more closely right now. Thank you for your feedback